main.pm

```diff
@@ -424,6 +424,7 @@ sub load_reboot_tests() {
         loadtest "installation/boot_encrypt.pm";
     }
     if (installyaststep_is_applicable) {
+        loadtest "installation/grub_test.pm";
```

Comment:
the whole file "grub_test.pm" should not be part of this PR which says "rename livecdreboot"